<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>背景抠图工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="src_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>430</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string>原图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="tar_label">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>430</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string>处理后的图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>640</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBG_pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>630</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>选择背景图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>150</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string>背景图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="cursorColor_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>680</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBGColor_pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>680</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>设置背景颜色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;抠图-&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>430</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>620</y>
      <width>430</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>3</width>
      <height>600</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>3</width>
      <height>600</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>430</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>3</width>
      <height>600</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>620</y>
      <width>430</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>20</y>
      <width>3</width>
      <height>600</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>150</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>3</width>
      <height>210</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>590</y>
      <width>150</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>3</width>
      <height>210</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>640</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>蓝底渐变</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>灰底渐变</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>红底渐变</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>白底</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>透明背景</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>从文件中选择...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>选择纯色背景...</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>645</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2.鼠标点击图片可选择颜色</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>659</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>304</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>640</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>646</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>659</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>或</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
